### PR TITLE
ISSUE - Update Dependencies, Fix Bug

### PR DESCRIPTION
**Pull request by vanard**

### CODE REVIEW

File: app/src/main/java/com/hypeloka/view/fragment/HypeFragment.kt [hunk 1]
```diff
@@ -155,15 +156,18 @@ class HypeFragment : Fragment() {
         view.post_chip_categories.setOnCheckedChangeListener { chipGroup, i ->
             val chip = chipGroup.findViewById<Chip>(i)
             if (chip != null){
-                if (chip.text.toString().toLowerCase().equals("all"))
+                if (chip.text.toString().toLowerCase() == "all")
                     hypeViewModel.selectedCategory.value = chip.text.toString().toLowerCase()
```
Comment (husentoding) [inline app/src/main/java/com/hypeloka/view/fragment/HypeFragment.kt]: kenapa pake `==` broh?

Comment (vanard) [reply app/src/main/java/com/hypeloka/view/fragment/HypeFragment.kt]: karena kotlin bruh

Comment (husentoding) [reply app/src/main/java/com/hypeloka/view/fragment/HypeFragment.kt]: itu ada toast "share pressed" pas click share, deletin dong

Comment (vanard) [reply app/src/main/java/com/hypeloka/view/fragment/HypeFragment.kt]: done

**Pull request merged**